### ISSUE TITLE
Why Is black screen, the video is saved

### ISSUE THREAD

**Issue by killsanghyuck** [hoang89yenthe/AndroidRTCRecorder_Libjingle_ver11139#3]
once i checked the stored files, it was a black screen.

**Comment by hoang89yenthe** [comment]
Yes, this is only a simple test project. Function record video is not working. You can record video using Everyplay.

**Comment by killsanghyuck** [comment]
thank you, Then, do you know how to use the recording feature in GLSurfaceView

**Comment by hoang89yenthe** [comment]
Unfortunately, I dont how to do this yet. If you have any idea, please inform me

**Comment by Skatic7** [comment]
@hoang89yenthe  @killsanghyuck 
Hello guys,
Just a question. Is this app working? Because I installed it and it only starts local stream, not remote android stream. App is not connecting two android devices. 
Thank you!

**Comment by hoang89yenthe** [comment]
Please refer pchab 's project. It's still working https://github.com/pchab
